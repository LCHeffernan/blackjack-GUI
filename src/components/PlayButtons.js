import React from "react";

const PlayButtons = () => (
  <div>
    <button className="play-button" type="button">
      Hit
    </button>
    <button className="play-button" type="button">
      Stand
    </button>
  </div>
);

export default PlayButtons;
