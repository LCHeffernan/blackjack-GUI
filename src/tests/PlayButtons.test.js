import React, { render } from "@testing-library/react";
import PlayButtons from "../components/PlayButtons";

describe("PlayButtons", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<PlayButtons />);
    expect(asFragment()).toMatchSnapshot();
  });
});
