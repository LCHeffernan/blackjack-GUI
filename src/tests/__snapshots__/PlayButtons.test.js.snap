// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayButtons renders correctly 1`] = `
<DocumentFragment>
  <div>
    <button
      class="play-button"
      type="button"
    >
      Hit
    </button>
    <button
      class="play-button"
      type="button"
    >
      Stand
    </button>
  </div>
</DocumentFragment>
`;
